content/livesync/connector/index.textile

```diff
@@ -0,0 +1,336 @@
+---
+title: Deploy
+meta_description: "The Ably Database Connector is available as a Docker container. Deploy it locally using Docker Compose, or on any cloud infrastructure that supports Docker images."
+product: livesync
+---
+
+The Database Connector uses the "outbox pattern":https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/transactional-outbox.html to broadcast application-specific change events from your database to frontend clients via "Ably Channels":/channels.
+
+When you update the data in your database, you record a change event transactionally in a dedicated "outbox table":/livesync/connector/database#outbox. The Database Connector detects new changes in the outbox and publishes them as messages on Ably Channels. Using the Models SDK, your client applications subscribe to these change events and update their local state with the new data.
+
+The Database Connector is available as a "Docker":/https://www.docker.com/ container image that can be deployed on any cloud infrastructure platform that supports running Docker container images.
```

Comment:
I don't think we want a leading slash on the link here, as it resolves to an incorrect relative path on the docs site: https://ably-docs-livesync-alph-pcjpdk.herokuapp.com/docs/https://www.docker.com

Reply:
Fixed in this [commit](https://github.com/ably/docs/pull/2082/commits/15a2be26866049471006c24c9e1e9094c5b53a85)